docs/lang/python.md

```diff
@@ -0,0 +1,105 @@
+# Building a Python Environment
+
+This guide covers configuring Python environments in envd. If you’re new to envd please read our [Tutorial](../get-started) and [build configuration guides](../build-envd) first.
+
+Let's begin 🐍!
+
+## Specifying Python
+
+The default language in envd is Python, thus there is no need to specify language. Or you can use `base` function to specify.
+
+```python title=build.envd
+def build():
+    base(os="ubuntu20.04", language="python")
+```
+
+## Conda packages
+
+You can install conda packages with `install.conda_packages` function. The following example installs `numpy` and `scipy`:
+
+```python title=build.envd
+def build():
+    base(os="ubuntu20.04", language="python")
+    install.conda_packages(name = [
+        "numpy",
+        "scipy",
+    ])
+```
+
+## PyPI packages
+
+You can install Python packages from PyPI with `install.python_packages` function. The following example installs `scikit-learn` and `matplotlib`:
+
+```python title=build.envd
+def build():
+    base(os="ubuntu20.04", language="python")
+    install.python_packages(name = [
+        "scikit-learn",
+        "matplotlib",
+    ])
+```
+
+envd uses system-wide [pip](https://pip.pypa.io/) to install Python packages in the previous example.
+
+If conda is enabled, you can also install Python packages from PyPI with `install.python_packags` function. The following example installs `numpy` and `scipy` with conda, then installs `scikit-learn` and `matplotlib` with pip:
```

Comment:
`then`? Is there an order here?

Reply:
The example here installs conda first but we do not limit the order in envd.

Reply:
I understand, but here using `then` implies the installation is in the order, which is not actually true in envd. In implementations, these two stages are merged together, right?

Reply:
I am not sure if I understand. Conda and pip are merged by the merge op. `then` is only used to describe the order in the example, not in the implementation.

Reply:
@VoVAllen Or do you have suggestions about the word?

Reply:
I prefer using `and`

Reply:
SGTM